infra/terraform/test-org/github/protection.tf

```diff
@@ -164,7 +180,7 @@ resource "github_branch_protection" "terraform-example-foundation" {
       "cla/google",
       "terraform-example-foundation-int-trigger-default (cloud-foundation-cicd)",
       "terraform-example-foundation-int-trigger-HubAndSpoke (cloud-foundation-cicd)",
-      "terraform-example-foundation-lint-trigger (cloud-foundation-cicd)",
+      "lint / lint (pull_request)",
```

Comment:
@bharathkkb - I'd rather leave this even if `disable_lint_yaml = true` as this is authoritative and a linter with the same name should be in-place.

Reply:
Yup sounds good